content/basics.article

```diff
@@ -1,130 +1,182 @@
-Packages, variables, and functions.
-Learn the basic components of any Go program.
+Пакеты, переменные и функции.
+Знакомимся с базовыми составляющими любой программы на Go.
+#en: Packages, variables, and functions.
+#en: Learn the basic components of any Go program.
 
 The Go Authors
 https://golang.org
 
-* Packages
+* Пакеты
+#en: * Packages
 
-Every Go program is made up of packages.
+Каждая программа на Go состоит из пакетов.
+#en: Every Go program is made up of packages.
 
-Programs start running in package `main`.
+Пакет `main` - это точка запуска программы.
+#en: Programs start running in package `main`.
 
-This program is using the packages with import paths `"fmt"` and `"math/rand"`.
+Программа, представленная в примере, импортирует два пакета с путями `"fmt"` и `"math/rand"`.
+#en: This program is using the packages with import paths `"fmt"` and `"math/rand"`.
 
-By convention, the package name is the same as the last element of the import path. For instance, the `"math/rand"` package comprises files that begin with the statement `package`rand`.
+Существует соглашение, по которому, имя пакета совпадает с последним элементом пути импорта. Например, пакет `"math/rand"` содержит файлы, которые начинаются с оператора `package`rand`.
```

Comment:
https://ru.wikipedia.org/wiki/%D0%9E%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%BE%D1%80_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)